src/batch/src/executor2/top_n.rs

```diff
@@ -97,27 +99,26 @@ impl BoxedExecutorBuilder for TopNExecutor {
             .map(OrderPair::from_prost)
             .collect();
         if let Some(child_plan) = source.plan_node.get_children().get(0) {
-            let child = source.clone_for_plan(child_plan).build()?;
-            return Ok(Box::new(
-                Self::new(
-                    child,
-                    order_pairs,
-                    top_n_node.get_limit() as usize,
-                    source.plan_node().get_identity().clone(),
-                )
-                .fuse(),
-            ));
+            let child = source.clone_for_plan(child_plan).build2()?;
+            return Ok(Box::new(Self::new(
+                child,
+                order_pairs,
+                top_n_node.get_limit() as usize,
+                source.plan_node().get_identity().clone(),
+                1024,
```

Comment:
Please don't use magic number here, we have predefined constants for default chunk size: `src/common/src/util/chunk_coalesce.rs:25`